aqua-examples/near-integration/README.md

```diff
@@ -183,20 +219,22 @@ Result:
 }
 ```
 
+In the output above listed the called function (`account_state`) with its arguments as well as the relay used for the call. So, you can observe the context of the function call. And there's a result of the call, of course. In our case it displays [the basic insformation](https://docs.near.org/tools/near-api-js/account#state) for our Near account.
```

Comment:
it should be 
*[the basic information]*

Reply:
good catch, thanks!